### PR TITLE
Proposed changes to ADAS signals spec. in compliance with SAE J3016

### PR DESCRIPTION
**Pull request by hongkicha**
Hi, I'm Hongki CHA, the Sponsor of SAE J3016.

I'd like to propose some changes to better comply with the latest SAE J3016.

I welcome further discussion.

Signed-off-by: Hongki CHA <cha8476@etri.re.kr>

### CODE REVIEW

File: spec/ADAS/ADAS.vspec [hunk 1]
```diff
@@ -1,3 +1,4 @@
+Signed-off-by: Hongki CHA <cha8476@etri.re.kr>
```
Comment (erikbosch) [inline spec/ADAS/ADAS.vspec:1]: Signed off shall be in commit, not in file.

**Comment by SebastianSchildt** [comment]
Content-wise I support this change. Thanks @hongkicha making this aligned better to the specific definitions from  J3016.

This also seems a candidate for backporting to the 4.x, potentially with the exclusion of the new `SAE_5_DISENGAGING` state, that has been added.

**Comment by erikbosch** [comment]
Meeting notes:

- Please review
- Sebastian: Makese sensor, maybe backport but adding SAE_5_disengage only to major/master

**Comment by erikbosch** [comment]
Hi @hongkicha.

There are some errors in Continuous Integration (CI) for your PRs. There are some guidelines at https://github.com/COVESA/vehicle_signal_specification/blob/master/BUILD.md that possibly could help. Feel free to reach our if you need help understanding the CI errors.

**Comment by hongkicha** [comment]
@erikbosch @SebastianSchildt Hi, this is Hongki. Thanks for your comments. We're having our Thanksgiving holiday in Korea until Tuesday. 
Considering the signed-off, I use GitHub Chrome directly, so I'm not sure how/where I can enter the command line. If necessary, I can set up my computer after I get back to my office.
I'm attending next week's COVESA AMM, and if possible, we can meet there and talk about how to proceed.

**Comment by erikbosch** [comment]
@hongkicha  -looking forward to meeting you. I believe there are some limitations if just using the Github web UI to create or modify commits compared to creating/modifying a commit on a local machine

**Comment by erikbosch** [comment]
Meeting notes:

- CI errors to be fixed
- Apart from that no blocker for merging for 5.0

**Comment by erikbosch** [comment]
Merge notes:

- Erik to assist making CI check success, then ok to merge

**Comment by erikbosch** [comment]
Merged by #672

**Pull request closed without merging**